packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -154,6 +154,52 @@ describe('Checkout', () => {
                 expect(addressBlock.exists()).toBe(false);
             });
         });
+
+        describe('hasCheckoutLoadedSuccessfully ::', () => {
+            it('should render the checkout form and not the error page when set to `true`', () => {
```

Comment:
If you can think of how to word the test name, it might be good to combine these two cases using `it.each`

Reply:
I think we had a similar conversation in a recent PR (don't remember if it was mine or somebody else's). In this case, when there are two test cases that are opposite of each other, I don't see the value in adding a `.each` and make the name a bit more complicated.

If we had a combination of various inputs in a test, in that case, I'd fully support the `.each`.

What do you think, @xander-marjoram?